themes/survey/fruity_twentythree/css/variations/theme_blueberry-rtl.css

```
@@ -11065,36 +11071,36 @@ body {
+  border-bottom-color: #dff3d6;
   border-bottom-color: rgba(0, 0, 0, 0.25);
 }
 .fruity_twentythree .popover.bottom > .arrow:after {
   border-bottom-color: #ffffff;
 }
 .fruity_twentythree .popover.left > .arrow {
-  border-right-color: rgb(222.5593257143, 242.902, 214.498);
+  border-right-color: #dff3d6;
   border-right-color: rgba(0, 0, 0, 0.25);
 }
 .fruity_twentythree .popover.left > .arrow:after {
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)